Tests/Integration/Microsoft365DSC/M365DSCIntegration.AAD.Create.Tests.ps1

```diff
@@ -32,6 +32,7 @@
                 {
                     DisplayName               = "AppDisplayName"
                     AvailableToOtherTenants   = $false
+                    Description                   = 'Test Description'
```

Comment:
Can you fix the indentation of the equals sign to match the other lines?

Reply:
Sorry for those indentation issues. Both sorted.